godot_project/project_workspace/workspace_context/create_object_parameters.gd

```diff
@@ -22,11 +22,12 @@ signal validate_bonds_requested(selection_only: bool)
 
 
 enum CreateModeType {
-	CREATE_ATOMS_AND_BONDS,
-	CREATE_SHAPES,
-	CREATE_FRAGMENT,
-	CREATE_VIRTUAL_MOTORS,
-	CREATE_ANCHORS_AND_SPRINGS,
+	CREATE_ATOMS_AND_BONDS     = 0,
+	CREATE_SHAPES              = 1,
+	CREATE_FRAGMENT            = 2,
+	CREATE_VIRTUAL_MOTORS      = 3,
+	CREATE_PARTICLE_EMITTERS   = 4,
+	CREATE_ANCHORS_AND_SPRINGS = 5,
```

Comment:
Is this stored on disk at any point?
If not we probably don't need to enforce a value for each.